### ISSUE TITLE
Potential mistake in your read.me

### ISSUE THREAD

**Issue by yuanggeng** [ChaoHuang2018/POLAR_Tool#2]
Thanks for sharing your great work! I have one question about how to input weights and biases of NN into POLAR. In your neural_network.py, it seems like you read the weights and biases for each neuron like: w,w,b,w,w,b... not w,w,w,w,w,...,b,b,b,.... Thanks for your time.

**Comment by ChaoHuang2018** [comment]
Exactly. Thanks for being interested in our work and pointing out the mistake in the document!

P.S., we are soon going to release a new version of POLAR that has a significant improvement on time efficiency.

Best,
Chao

From: yuanggeng ***@***.***>
Date: Tuesday, 4 April 2023 at 17:44
To: ChaoHuang2018/POLAR_Tool ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ChaoHuang2018/POLAR_Tool] Potential mistake in your read.me (Issue #2)

Thanks for sharing your great work! I have one question about how to input weights and biases of NN into POLAR. In your neural_network.py, it seems like you read the weights and biases for each neuron like: w,w,b,w,w,b... not w,w,w,w,w,...,b,b,b,.... Thanks for your time.

**Comment by yuanggeng** [comment]
Thank you so much! Can I double check what is the time step you set in each control step maybe in the single pendulum example? Can you explain the differene when you deal with ODE and DDE? Thank you : )

**Comment by ChaoHuang2018** [comment]
In the single pendulum example, the control step size is 0.05.

For DDE encoding, please refer to the mountain car example (POLAR_Tool/examples/Mountain_Car/mountain_car.cpp). Simply speaking, the main difference is the construction of the dynamics: use DDE() to construct DDE-based dynamics (e.g., line 42, mountain_car.cpp), and use ODE() to construct ODE-based dynamics (e.g., line 26, single_pendulum.cpp).
